index.html

```diff
@@ -46,5 +42,189 @@ <h3 class="logo">John Doe </h3>
     <div class="bckimg">
         <img class="sideimg"src="images/illustration-header-mobile.png" alt="illustration">
     </div>
+    
+     <!--Second Branch: Recent Work and About me Page-->
+<section class="workSection">
+    <div>
+    <p class="workSecPara">My Recent Works</p>
+    
+    <img class="workSectionRec" src="images/rec2.png" alt="Rec2">
+</div>
+
+<div>
+    <img class="workSectionImg"src="images/Placeholder.png" alt="Section Image">
+    <p class="wrkSecHeader">Multi-Post Stories</p>
+    <p class="wrkSecPara">A daily selection of privately personalized 
+        reads; no accounts or sign-ups required. 
+        has been the industry's standard 
+        dummy text ever since the 1500s, when an 
+        unknown printer took a standard dummy 
+        text.</p>
+        <ul class="language">
+            <li class="css"><a href="">css</a></li>
+            <li class="html"><a href="">html</a></li>
+            <li class="bootstrap"><a href="">bootstrap</a></li>
+            <li class="ruby"><a href="">ruby</a></li>
+        </ul>
+        <button class="btnSeeProject"><a href="#">See Project</a></button>
```

Comment:
- Great job wrapping your `class="language"` in an unordered list 🤗. I would like to suggest that adding an anchor tag to each `</li>` is not necessary at this point because these lists aren't meant to redirect you anywhere else. I think you should consider removing them 👍🏽 

- The same applies to the button. It should be this 👉🏽 `<button class="btnSeeProject">See Project</button>` 🍒 

_Why don't we apply these changes to all tags that you may have mistakenly added them to?? 😄_